src/components/shared/StreamingPayments/utils.ts

```diff
@@ -45,50 +44,15 @@ export const calculateToCurrency = async ({
   return new Decimal(balanceInWeiToEth).mul(currentPrice ?? 0);
 };
 
-const calculateStreamsForLast30Days = ({
-  startTimestamp,
-  endTimestamp,
-  currentTimestamp,
-  interval,
-  totalClaimedAmount,
-}: {
-  startTimestamp: number;
-  endTimestamp: number;
-  currentTimestamp: number;
-  interval: number;
-  totalClaimedAmount: number;
-}) => {
-  const intervalInDays = interval / 24 / 3600;
+const calculateStreamsForLast30Days = (
```

Comment:
Just dropping in to say this name feels a bit misleading now given the change in implementation, maybe `calculateCurrentStreamingRate` or something?

Reply:
Meybe just calculate ActiveStreaming? 😁